nerdtribe/mr-binge
# Routing

## Explanation of work done and why:


Add routes to tv and movies pages

## How to (manually) test:

1. Fetch `git fetch -p && git checkout -b feature/routing-fix origin/feature/routing-fix`
2. Serve `yarn electron:serve`
3. Observe a navbar with two buttons for tv and movies and the current page listed below. Clicking on each button toggles the view.
4. Should be no console errors.

## Checklist:
- [x] **Closes**  #16 Routing
